lib/spack/spack/cmd/common/arguments.py

```diff
@@ -81,7 +81,7 @@ def _specs(self, **kwargs):
 _arguments['module_type'] = Args(
     '-m', '--module-type',
     choices=spack.modules.module_types.keys(),
-    default=spack.modules.module_types.keys()[0],
+    action='append',
     help='type of module files [default: %(default)s]')
```

Comment:
Good catch on the help message. I changed the action to `append` so that one can do:
```
spack module refresh -m tcl -m lmod ...
```
The default is the one you proposed (all the module files that are known) except that it's set in the method `module`.